inc/knowbaseitem.class.php

```diff
@@ -1262,61 +1309,67 @@ static function getListRequest(array $params, $type = 'search') {
                                    /* 8 */   "/\)/",
                                    /* 9 */   "/\-/"];
                   $contains = preg_replace($search1, "", $params["contains"]);
-                  $addwhere = '';
+                  $ors = [
+                     "glpi_knowbaseitems.name ".Search::makeTextSearch($contains),
```

Comment:
This will produce raw SQL.
```
$search_inst = new Search(new KnowbaseItem(), []);
$ors = [
   'glpi_knowbaseitems.name' => ['LIKE', $search_inst->makeTextSearchValue($contains)],
   'glpi_knowbaseitems.answer' => ['LIKE', $search_inst->makeTextSearchValue($contains)],
];
```